docs/tutorial.rst

```diff
@@ -563,3 +569,101 @@ and manually updating the action set to use artifacts created by the
 previous action set can be cumbersome. In situations like this, it is
 useful to instead use ``kanctl``. To learn how to leverage ``kanctl`` to
 create action sets, see :ref:`architecture` .
+
+Blueprint Development
+=====================
+
+To start with Blueprint Development we need to be aware of some terms such as
+Actions, Phase, KanisterFunctions, Objects which has already been covered in
+this tutorial.
+
+Define and add ``actions`` for the blueprint as per your application. Each action
+performs a set of tasks. For example, backup, restore, and delete.
```

Comment:
actions can be user defined right? Not necessarily backup/restore/delete. Can we specify that?